apps/condo/domains/ticket/schema/Ticket.test.js

```diff
@@ -3384,163 +3392,460 @@ describe('Ticket', () => {
 
-                expect(message).toBeUndefined()
-            })
+                    const [organization] = await createTestOrganization(admin)
+                    const [property] = await createTestProperty(admin, organization)
+                    const unitName = faker.random.alphaNumeric(5)
+                    const unitType = FLAT_UNIT_TYPE
+                    const { phone } = residentClient.userAttrs
 
-            test('dont send sms if resident user create ticket', async () => {
-                const residentClient = await makeClientWithResidentUser()
+                    await createTestResident(admin, residentClient.user, property, {
+                        unitName,
+                        unitType,
+                    })
 
-                const [organization] = await createTestOrganization(admin)
-                const [property] = await createTestProperty(admin, organization)
-                const unitName = faker.random.alphaNumeric(5)
-                const unitType = FLAT_UNIT_TYPE
-                const { phone } = residentClient.userAttrs
+                    await createTestTicket(residentClient, organization, property, {
+                        unitName,
+                        unitType,
+                    })
 
-                await createTestResident(admin, residentClient.user, property, {
-                    unitName,
-                    unitType,
-                })
+                    const messageWhere = { phone, type: TRACK_TICKET_IN_DOMA_APP_TYPE }
+                    const message = await Message.getOne(admin, messageWhere)
 
-                await createTestTicket(residentClient, organization, property, {
-                    unitName,
-                    unitType,
+                    expect(message).toBeUndefined()
                 })
+            })
 
-                const messageWhere = { phone, type: TRACK_TICKET_IN_DOMA_APP_TYPE }
-                const message = await Message.getOne(admin, messageWhere)
+            describe('push to employees', () => {
+                describe('by ticket visibility', () => {
+                    let propertyInScope, propertyOutOfScope,
+                        employeeInScopeClient, employeeOutOfScopeClient,
+                        employeeInScope, employeeOutOfScope,
+                        employeeRole, testScope
+
+                    beforeAll(async () => {
+                        employeeInScopeClient = await makeClientWithNewRegisteredAndLoggedInUser()
+                        employeeOutOfScopeClient = await makeClientWithNewRegisteredAndLoggedInUser()
+
+                        const [defaultRole] = await createTestOrganizationEmployeeRole(admin, organization, {
+                            canManageTickets: true,
+                        })
+                        employeeRole = defaultRole
+
+                        const [employee1] = await createTestOrganizationEmployee(admin, organization, employeeInScopeClient.user, employeeRole)
```

Comment:
it will be shorter
```
employeeInScopeClient = await makeClientWithNewRegisteredAndLoggedInUser()
employeeOutOfScopeClient = await makeClientWithNewRegisteredAndLoggedInUser()

[defaultRole] = await createTestOrganizationEmployeeRole(admin, organization, {
    canManageTickets: true,
});

[employee1] = await createTestOrganizationEmployee(admin, organization, employeeInScopeClient.user, employeeRole);
[employee2] = await createTestOrganizationEmployee(admin, organization, employeeOutOfScopeClient.user, employeeRole);

[property1] = await createTestProperty(admin, organization);
[property2] = await createTestProperty(admin, organization);

[scope] = await createTestPropertyScope(admin, organization)

await createTestPropertyScopeProperty(admin, scope, propertyInScope)
await createTestPropertyScopeOrganizationEmployee(admin, scope, employeeInScope)

const syncRemoteClientPayload = {
    devicePlatform: DEVICE_PLATFORM_ANDROID,
    appId: APP_MASTER_ID_ANDROID,
}
await syncRemoteClientWithPushTokenByTestClient(employeeInScopeClient, syncRemoteClientPayload)
await syncRemoteClientWithPushTokenByTestClient(employeeOutOfScopeClient, syncRemoteClientPayload)

```